File: scratch-collect.Model/User/EditProfileRequest.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace scratch_collect.Model.Requests
{
    public class EditProfileRequest
    {
        public string Email { get; set; }

        public string Username { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Address { get; set; }

    }
}

File: scratch-collect.MobileOld/lib/modules/profile/models/edit_profile_request.dart
class EditProfileRequest {
  int? id;
  String? email;
  String? username;
  String? firstName;
  String? lastName;
  String? address;

  EditProfileRequest(
      {this.id,
      this.email,
      this.username,
      this.firstName,
      this.lastName,
      this.address});

  EditProfileRequest.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    email = json['id'];
    username = json['username'];
    firstName = json['firstName'];
    lastName = json['lastName'];
    address = json['address'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};

    data['id'] = id;
    data['email'] = email;
    data['username'] = username;
    data['firstName'] = firstName;
    data['lastName'] = lastName;
    data['address'] = address;

    return data;
  }
}
